tests/test_elastic.py

```diff
@@ -478,3 +480,56 @@ def test_connection_error(self):
         )
         self.user.save()
         self.project.save()
+
+
+class TestSearchMigration(SearchTestCase):
+    """
```

Comment:
Prefer long names to docstrings for tests: http://cosdev.readthedocs.org/en/latest/process/testing.html#general-testing-guidelines